lunatik.h

```diff
@@ -193,18 +177,20 @@ static inline void lunatik_setclass(lua_State *L, const lunatik_class_t *class)
 	lua_setiuservalue(L, -2, 1); /* pop class */
 }
 
-static inline void lunatik_setobject(lunatik_object_t *object, const lunatik_class_t *class, bool sleep)
+static inline void lunatik_setobject(lunatik_object_t *object, const lunatik_class_t *class, bool sleep, bool single)
```

Comment:
I'm rethinking this; this a good reason for having a single `flags` parameter.. otherwise, we will need a new argument for every new decision we need.. it doesn't scale.. what do you think?